### PR TITLE
Update REAMDEs for iOS and Android to have more details

### PR DESCRIPTION
**Pull request by bplattenburg**
Updates the iOS README and Android README to spell out more details of the Ditto app secrets file setup, aligning the two more closely.

Removes all references to `DITTO_OFFLINE_TOKEN`  in docs and code to simplify integration, as that will not be available out of the box except to customers where we can also provide more specific guidance.

### CODE REVIEW

**Comment by bplattenburg** [comment]
I'm going to take a little more time and flesh the READMEs for both platforms with a big more explicit details around when and why to use the different fields, marking this as a draft for now

**Comment by bplattenburg** [comment]
> I'm going to take a little more time and flesh the READMEs for both platforms with a big more explicit details around when and why to use the different fields, marking this as a draft for now

This is now done and ready to review

File: Android/README.md [hunk 1]
```diff
@@ -18,17 +18,18 @@ Android version of the Chat App Demo
...
     
     DITTO_APP_ID = "replace with your app id"
-    DITTO_PLAYGROUND_TOKEN = "replace with your playground token"
-    DITTO_AUTH_PASSWORD = "replace with your auth password"
-    DITTO_AUTH_PROVIDER = "replace with your auth provider"
-    DITTO_OFFLINE_TOKEN = "replace with your offline license token"
+    DITTO_PLAYGROUND_TOKEN = "replace with your playground token if applicable"
+    DITTO_AUTH_PASSWORD = "replace with your auth password if applicable"
+    DITTO_AUTH_PROVIDER = "replace with your auth provider if applicable"
+    DITTO_OFFLINE_TOKEN = "replace with your offline license token if applicable"
```
[the remark this answers is not in the document]
Comment (rajramsaroop) [reply Android/README.md:32]: I just noticed in my portal that the offline token is no longer there, so I think moving forward we move any mention/relation to offline token.

That would also mean updating the rest of this gradle file as well as updating the references in the code sample.

File: iOS/README.md [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+
+5. Edit `.env` to add environment variables as in the following example, substituting your own values as needed (based on your app configuration in the portal you will only need a subset of these values):
 ```
-APP_ID=a01b2c34-5d6e-7fgh-ijkl-8mno9p0q12r3
-DITTO_OFFLINE_TOKEN=a01b2c34-5d6e-7fgh-ijkl-8mno9p0q12r3
+    DITTO_APP_ID = "replace with your app id"
+    DITTO_PLAYGROUND_TOKEN = "replace with your playground token"
+    DITTO_AUTH_PASSWORD = "replace with your auth password"
+    DITTO_AUTH_PROVIDER = "replace with your auth provider"
+    DITTO_OFFLINE_TOKEN = "replace with your offline license token"
```
Comment (rajramsaroop) [inline iOS/README.md:17]: Similar comment regarding Android. Remove references to the offline token (in the readme and in the code) - I don't even see it in my DItto portal now.

**Review by rajramsaroop — CHANGES_REQUESTED** [review]

File: iOS/README.md (hunk 2 above)
Comment (rajramsaroop) [reply iOS/README.md:17]: Based on the Android comments, it sounds like the offline token stays, but some tweaking to the language might be needed.

File: Android/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (magicwave) [reply Android/README.md:32]: @bplattenburg - you have that right. We can remove reference to offline token. Note that the env.template file is retained in the repo, while the (copied) .env file containing credentials is gitignored.

File: iOS/README.md (hunk 2 above)
Comment (magicwave) [reply iOS/README.md:17]: I think we should remove the offline token statement from the README and the env.template. In cases of customers who get an offline token license, we can support them directly if needed, but we don't offer offline tokens for self serve customers.

Comment (bplattenburg) [reply iOS/README.md]: Based on this conversation, I've removed all references to the offline token in the Readme, config files, and code.

**Pull request merged**